services/webhooks2tasks/src/handlers/githubPush.js

```diff
@@ -19,6 +19,15 @@ async function githubPush(webhook: WebhookRequestData, project: Project) {
     const branchName = body.ref.toLowerCase().replace('refs/heads/','')
     const sha = body.after
 
+    const skip_deploy = body.commits.map( commit => Array.isArray(commit.message.match(/skip deploy|deploy skip/i)))
+                                    .reduce( function(a,i,x,z) { return a||i })
+    if (skip_deploy) {
+      sendToLagoonLogs('info', project.name, uuid, `${webhooktype}:${event}:handled`, meta,
```

Comment:
can we change the message type from `${webhooktype}:${event}:handled` to `${webhooktype}:${event}:skipped`

Reply:
also if we do that, we have to update logs2slack that it sends a slack message for these type of events